ldms/src/core/ldms.h

```diff
@@ -1451,7 +1560,12 @@ struct ldms_stream_recv_data_s {
 	uint32_t data_len;
 	const char *name; /* stream name */
 	const char *data; /* stream data */
-	json_entity_t json; /* json entity */
+
+	json_entity_t json; /* json entity (LDMS_STREAM_JSON type) */
```

Comment:
Why is the json field present at the same time as the the avro fields? Shouldn't that be a union to make it clear that it will be one or the other?

How would the user of the API go about freeing this struct? Is there a helper function to do that?